### PR TITLE
Update description for PackedElementField

### PR DESCRIPTION
**Pull request by jtbandes**
**Public-Facing Changes**
Updated description for PackedElementField

### CODE REVIEW

**Review by defunctzombie — APPROVED** [review]

File: schemas/jsonschema/Grid.json [hunk 1]
```diff
@@ -112,7 +112,7 @@
       "items": {
         "$comment": "Generated from PackedElementField by @foxglove/message-schemas",
         "title": "PackedElementField",
-        "description": "List of fields for every element in a byte array",
+        "description": "Defines a field within packed data. Commonly used to represent a field of a single element within a byte array containing multiple packed elements.",
```
Comment (esthersweon) [inline schemas/jsonschema/Grid.json:115]: Can remove "Defines" from beginning of every version of this schema. Don't think the other schemas are formatted this way.

Comment (esthersweon) [reply schemas/jsonschema/Grid.json:115]:
```suggestion
  "description": "A field present within each element in a byte array of packed elements.",
```

**Review by esthersweon — CHANGES_REQUESTED** [review]
Suggested changes to description.

File: schemas/jsonschema/Grid.json (hunk 1 above)
Comment (esthersweon) [reply schemas/jsonschema/Grid.json:115]: Or maybe even "each packed element in a byte array".

**Review by esthersweon — APPROVED** [review]

**Pull request merged**